### DISCUSSION TITLE
How can I access multiple values in cell definition?

### DISCUSSION THREAD

**Discussion by jimsheen** [TanStack/table#4148] [Q&A] [answered]
Hi, thanks for the library!

In the example below I want to render name and jobTitle in the same cell - is this possible? I know the accessorFn can only return primitives. 

I think this used to work in beta because the return type of `getValue` was `any` (and it rendered the correct value). However,  now the type is unknown (which I believe is a separate issue - https://github.com/TanStack/table/issues/4142 )

```
id: 'info',
      accessorFn: (row) => row,
      cell: ({ getValue }) => (
        <>
          <h1>{getValue().name}</h1>
          <h2>{getValue().jobTitle}</h2>
        </>
      ),
```

**Comment by KevinVandy** [comment, marked as answer]
Look at cell.row.original
